zh_CN/kicad.po

```diff
@@ -11252,11 +11237,11 @@ msgid "Save Workbook"
 msgstr "保存工作簿"
 
 #: eeschema/sim/sim_plot_frame_base.cpp:35
-msgid "Save as Image"
+msgid "Save as image"
```

Comment:
What is the reasoning to change the capitalization of the word image?

Reply:
This kicad.po file was reproduced from the kicad-5.1.1 source code. Is the source code changed?